src/main/java/refactoringml/App.java

```diff
@@ -251,15 +253,13 @@ private void processCommit(RevCommit currentCommit, GitHistoryRefactoringMiner m
 			Set<ImmutablePair<String, String>> jGitRenames = getJGitRenames(entries);
 			processMetrics.collectMetrics(currentCommit, superCommitMetaData, allRefactoringCommits, entries, refactoringRenames, jGitRenames);
 			long startTimeTransaction = System.currentTimeMillis();
-			db.commit();
+			t.commit();
 			log.debug("Committing the transaction for commit " + commitHash + " took " + (System.currentTimeMillis() - startTimeTransaction) + " milliseconds.");
 		} catch (Exception e) {
 			exceptionsCount++;
 			log.error("Unhandled exception when collecting commit data for commit: " + commitHash + createErrorState(commitHash, project), e);
 			db.rollback(createErrorState(commitHash, project));
```

Comment:
Is the rollback, still working as intended?

Reply:
Yes i tested it by debugging, maybe an idea for the future to write a test for it though